pkgs/top-level/all-packages.nix

```diff
@@ -28124,8 +28124,10 @@ with pkgs;
     jdk = jdk8;
   };
 
-  rstudio-server = callPackage ../applications/editors/rstudio/server.nix {
+  rstudio-server = callPackage ../applications/editors/rstudio {
+    inherit (stdenv) mkDerivation;
     jdk = jdk8;
+    server = true;
```

Comment:
this server argument probably needs to get passed to the wrapper as well, although perhaps merging the two wrapper files is coming in the next commit.

Reply:
Yeah sorry just ran out of time yesterday :).